compositor_render/src/transformations/web_renderer/http_client.rs

```diff
@@ -0,0 +1,40 @@
+use bytes::Bytes;
+use compositor_common::scene::Resolution;
+use reqwest::blocking::Client;
+use serde::Serialize;
+
+pub struct HttpClient {
+    port: u16,
+    client: Client,
+}
+
+impl HttpClient {
```

Comment:
http client is sth that would have a method like get/post. This is a wrapper that is doing sth specific and it should be named appropriately .e.g  ElectronApi, WebRenderererClient, WebRenderererHttpClient